test/__tests__/user_transformation.test.js

```diff
@@ -45,988 +59,1132 @@ const getfetchResponse = (resp, url) =>
     responseInit(url)
   );
 
-const possibleEnvs = ["true", "false", "no_value", "some_random_value"];
-possibleEnvs.forEach(envValue => {
-  describe("User transformation", () => {
-    const OLD_ENV = process.env;
-    beforeEach(() => {
-      jest.resetAllMocks();
-      process.env = { ...OLD_ENV };
+describe("User transformation", () => {
+  beforeEach(() => {
+    jest.resetAllMocks();
+  });
+  afterAll(() => {});
+
+  it(`Simple ${name} Test`, async () => {
```

Comment:
There is a lot of repetition in the code below. Making it hard to read and maintain. Would it make sense to follow a similar pattern to Go's table-driven tests?

An array with:
[{
inputData
expectedData
code
}]```

Timeouts and HTTP errors can be move to another `Describe`